exercises/finale-workflow/Workflow/CertificateGeneratorWorkflow.cs

```diff
@@ -0,0 +1,17 @@
+namespace Temporalio.Finale.Workflow;
+using Temporalio.Workflows;
+
+[Workflow]
+public class CertificateGeneratorWorkflow
+{
+    [WorkflowRun]
+    public async Task<string> RunAsync(string name)
+    {
+        // The CertificateGeneratorWorkflow Workflow calls the CreatePdf Activity. 
+        // CreatePdf is the Activity Type defined in the implementation of the Java Activity code.
+        return await Workflow.ExecuteActivityAsync<string>("CreatePdf", new object[] { name }, new()
```

Comment:
You could make a typed interface, e.g. `IJavaActivities` that has a `string CreatePdf(string name)` call, and then you get type safety. This is usually better than passing strings around.